src/structures/Message.js

```diff
@@ -259,6 +260,47 @@ class Message {
   }
 
   /**
+   * Creates a reaction collector.
+   * @param {ReactionCollectorFilter} filter The filter to apply.
+   * @param {ReactionCollectorOptions} [options={}] Options to send to the collector.
+   * @returns {ReactionCollector}
+   * @example
+   * // create a reaction collector
+   * const collector = message.createCollector(
+   *  (reaction, user) => reaction.emoji.id === '👌' && user.id === 'someID',
+   *  { time: 15000 }
+   * );
+   * collector.on('reaction', r => console.log(`Collected ${r.emoji.name}`));
+   * collector.on('end', collected => console.log(`Collected ${collected.size} items`));
+   */
+  createCollector(filter, options = {}) {
```

Comment:
Even though there likely won't be more collectors for messages, I feel that it would be less ambiguous if you changed the method name to `createReactionCollector`